### DISCUSSION TITLE
How to launch on foreground

### DISCUSSION THREAD

**Discussion by navalex** [godotengine/godot-vscode-plugin#765] [Q&A] [answered]
Hey,

I'm trying your extension, and everything is really cool, except that my game is launching in background when I press F5. I tried to find something in extension options, but found nothing related.

Any idea about why does it behave like that ?

Godot version: 4.3
Platform: Windows 11

**Comment by DaelonSuzuka** [comment, marked as answer]
Existing issue: https://github.com/godotengine/godot-vscode-plugin/issues/692

**Reply by navalex** [reply]
Thanks ! So I guess it's gonna stay like this for now ? :/

**Reply by DaelonSuzuka** [reply]
That appears to be the case, yeah. This behavior doesn't appear to be consistent between users and I'm not certain there's a way to control this from inside VSCode anyways.
